bundle/Command/Validators.php

```diff
@@ -52,6 +52,19 @@ public static function validateLanguageCode(?string $languageCode = null): strin
         return $languageCode;
     }
 
+    public static function validateDesignType(?string $designType = null): string
+    {
+        if (empty($designType)) {
```

Comment:
`if ($designType === null) {` ? :)

Reply:
Sure, I did it like this to be consistent. Ok then to change the other if clauses (in a separate commit) as well?

Reply:
Ah, I didn't realize there are other instances of `empty` here. I'll leave the decision to you, both is fine since there are other places where it's used.

Reply:
Let's keep it as is, and we can clean up in some future PR :)

Reply:
:+1: